viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1569,10 +1570,12 @@ export class Cognite3DViewer {
       const camera = this.cameraManager.getCamera();
       TWEEN.update(time);
       this.recalculateBoundingBox();
-      this._activeCameraManager.update(
-        this.cameraManagerClock.getDelta(),
-        this._updateNearAndFarPlaneBuffers.combinedBbox
-      );
+
+      if (this._activeCameraManager instanceof FlexibleCameraManager) {
+        this._activeCameraManager.updateInnerBoundingBox(this._boundingBoxes.innerBoundingbox);
```

Comment:
I don't like that we depend on `FlexibleCameraManager` like this here. But I suppose you don't either :sweat_smile:

Reply:
I think this is the best we can no with the current architecture